website/docs/user_attributes.md

```diff
@@ -0,0 +1,166 @@
+| **OIDC claim location** | Userinfo endpoint                                                     |
+|              **origin** | Provided by user's Identity Provider                                  |
+|             **changes** | Yes                                                                   |
+|        **multiplicity** | Single-valued                                                         |
+|        **availability** | Always                                                                |
+|             **example** | _Legal Name 1_                                                        |
+|               **notes** | -                                                                     |
+|              **status** | Stable                                                                |
+
+### 8. Legal Person Identifier
+
+|          attribute name | Legal Name                                                                                    |
+| ----------------------: | :-------------------------------------------------------------------------------------------- |
+|         **description** | The user's legal person identifier                                                            |
+|   **SAML Attribute(s)** | `http://eidas.europa.eu/attributes/legalperson/LegalPersonIdentifier` (LegalPersonIdentifier) |
+|          **OIDC scope** | `legal_profile`                                                                               |
+|       **OIDC claim(s)** | `legal_person_identifier`                                                                     |
+| **OIDC claim location** | Userinfo endpoint                                                                             |
+|              **origin** | Provided by user's Identity Provider                                                          |
+|             **changes** | Yes                                                                                           |
+|        **multiplicity** | Single-valued                                                                                 |
+|        **availability** | Always                                                                                        |
+|             **example** | _CA/CA/UniqueId1_                                                                             |
+|               **notes** | -                                                                                             |
+|              **status** | Stable                                                                                        |
+
+### 9. Legal Address
+
+|          attribute name | Legal Name                                                                                      |
+| ----------------------: | :---------------------------------------------------------------------------------------------- |
+|         **description** | The user's legal address                                                                        |
```

Comment:
description: The current address for the natural person as registered with the eIDAS Member State authority.